openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -101,15 +101,32 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
                     </div>
                     <p class="setting-help">{% trans "The date and time when students can no longer submit responses." %}</p>
                 </li>
-                <li id="openassessment_submission_image_wrapper" class="field comp-setting-entry">
+                <li id="openassessment_submission_file_wrapper" class="field comp-setting-entry">
                     <div class="wrapper-comp-setting">
-                        <label for="openassessment_submission_image_editor" class="setting-label">{% trans "Allow Image Responses"%}</label>
-                        <select id="openassessment_submission_image_editor" class="input setting-input" name="image submission">
-                            <option value="0">{% trans "False"%}</option>
-                            <option value="1" {% if allow_file_upload %} selected="true" {% endif %}>{% trans "True"%}</option>
+                        <label for="openassessment_submission_upload_selector" class="setting-label">{% trans "Allow File Upload"%}</label>
+                        <select id="openassessment_submission_upload_selector" class="input setting-input" name="upload submission">
+                            <option value="none">{% trans "None"%}</option>
+                            <option value="image" {% if file_upload_type == "image" %} selected="true" {% endif %}>{% trans "Image File"%}</option>
+                            <option value="file" {% if file_upload_type == "file" %} selected="true" {% endif %}>{% trans "PDF or Image File"%}</option>
+                            <option value="custom" {% if file_upload_type == "custom" %} selected="true" {% endif %}>{% trans "Custom File Types"%}</option>
                         </select>
                     </div>
-                    <p class="setting-help">{% trans "Specify whether students can submit an image file along with their text response." %}</p>
+                    <p class="setting-help">
+                        {% trans "Specify whether students can submit a file along with their text response. 'Image' option allows JPG, GIF and PNG files. 'PDF or Image' option allows PDF and images. The 'Custom File Types' option allowes files with white listed extensions to be uploaded (use this option with caution)." %}
+                    </p>
```

Comment:
typo in "allowes"
I'd suggest, 
Select **Image** to allow JPG, GIF, or PNG files. Select **PDF or Image** to allow PDF files and images. Select **Custom File Types** to allow files with extensions that you specify in the **White List**. (Use this option with caution.)